### DISCUSSION TITLE
Are the embeddings extracted by the pyannote/embedding xvectors?

### DISCUSSION THREAD

**Discussion by PhilipAmadasun** [pyannote/pyannote-audio#1765] [Q&A]
Are the embeddings extracted from the pyannote-embedding x-vectors?

**Comment by hbredin** [comment]
Not exactly the same x-vectors as in [the original paper](https://www.danielpovey.com/files/2018_icassp_xvectors.pdf) but a variant of it.

**Comment by PhilipAmadasun** [comment]
Thanks for the reply. Why a variant? Does the variant provide a better
representation of voice features?
